### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by metric152**
updated the readme to show how to retrieve the string data from the message response

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -172,8 +172,8 @@ ws.on('open', function open() {
   ws.send('something');
 });
 
-ws.on('message', function message(data) {
-  console.log('received: %s', data);
+ws.on('message', function message(data, isBinary) {
+  console.log('received: %s', isBinary ? data : data.toString());
```
Comment (lpinca) [inline README.md:176]: This is not needed `data.toString()` is called automatically.

**Review by lpinca — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (metric152) [reply README.md:176]: When I tried it it returned “Buffer <“ and a bunch of numbers. Maybe it’s the version of node I’m using : 16. I had to use “toString” to access the data.

Comment (lpinca) [reply README.md:176]: It's not the Node.js version. See the string substitution (`%s`) in the message.

https://runkit.com/5ea585d2c57dcf0013d0bdc2/66c4fb24dc86270008e51a66

**Review by OKEAMAH — APPROVED** [review]
Good

**Comment by lpinca** [comment]
Thank you anyway.

**Pull request closed without merging**